src/styles/_components/_tabs.scss

```diff
@@ -0,0 +1,33 @@
+$tabs-height: 36px;
+
+@mixin tabs() {
+    width: 100%;
+    height: $tabs-height;
+
+    &__wrapper {
+        @include flex();
+        @include align-items-center();
+        position: relative;
+        width: 100%;
+        height: $tabs-height;
+        background-color: color('buttons', 'disabled');
```

Comment:
Shouldn't it be better to have a color for tabs specifically? When we """eventually""" change the buttons' colours, it will affect the tabs unnecessarily :)